docker-compose.yml

```diff
@@ -1,26 +1,24 @@
-version: "2"
+version: "3"
 
 services:
   ####################### Keycloak Definition #######################
-  keycloak:
-    restart: always
-    container_name: mds_keycloak
-    image: jboss/keycloak
-    environment:
-      - KEYCLOAK_USER=admin
-      - KEYCLOAK_PASSWORD=admin
-      - KEYCLOAK_IMPORT=/tmp/keycloak-local-realm.json
-      - DB_VENDOR=h2
-    volumes:
-      - ./settings/keycloak/keycloak-local-realm.json:/tmp/keycloak-local-realm.json
-      - ./settings/keycloak/keycloak-local-user.sh:/tmp/keycloak-local-user.sh
-    ports:
-      - "8080:8080"
-      - "8443:8443"
-    networks:
-      - mds
-    # logging:
-    #     driver: none
+  # keycloak:
```

Comment:
Any reason why this is commented out? Should it be removed?

Reply:
This was commented out as I felt like our docker-compose was getting pretty full. But seriously, we might need to consider a different approach for how we manage local services.